traffic_portal/test/integration/PageObjects/SideNavigationPage.po.ts

```diff
@@ -157,6 +158,11 @@ export class SideNavigationPage extends BasePage{
         await browser.actions().mouseMove(this.lnkServiceCategories).perform();
         await browser.actions().click(this.lnkServiceCategories).perform();
     }
+    async NavigateToCertExpirationsPage(){
+        await browser.wait(ExpectedConditions.visibilityOf(this.lnkCertExpirations), 2000);
+        await browser.actions().mouseMove(this.lnkCertExpirations).perform();
+        await browser.actions().click(this.lnkCertExpirations).perform();
```

Comment:
I don't think the part about moving the mouse is under test here, so we can just do this on one line with
```
return this.lnkCertExpirations.click();
```
This method is also (like most others) missing a return type (should be `Promise<void>`), and IMO ought to have a visibility modifier (`public`) and jsDoc comment (but neither of those are enforced, so don't feel like you have to do them - return type though is something I've been requesting on all callables, if not actually enforced).